### PR TITLE
DOC-251-Nested-Cluster

### PR DESCRIPTION
**Pull request by Siboney1**
Nested Cluster and Quickstart page

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/854#" title="Author self-approved">Siboney1</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by spectro-prow** [comment]
Hi @Siboney1. Thanks for your PR.

I'm waiting for a [spectrocloud](https://github.com/orgs/spectrocloud/people) member to verify that this patch is reasonable to test. If it is, they should reply with `/ok-to-test` on its own line. Until that is done, I will not automatically test new commits in this PR, but the usual testing commands by org members will still work. Regular contributors should [join the org](https://github.com/orgs/spectrocloud/people) to skip this step.

Once the patch is verified, the new status will be reflected by the `ok-to-test` label.

I understand the commands that are listed [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Comment by Siboney1** [comment]
Includes Nested Cluster and Quickstart

**Review by TylerGillson — COMMENTED** [review]
Couple of minor nits... looks great overall though!

File: content/docs/04-clusters/4-nested-clusters.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+import Tabs from 'shared/components/ui/Tabs';
+import InfoBox from 'shared/components/InfoBox';
+import WarningBox from 'shared/components/WarningBox';
+import PointsOfInterest from 'shared/components/common/PointOfInterest';
+import Tooltip from "shared/components/ui/Tooltip";
+
+
+# Nested Clusters Overview
+
+Nested Clusters are Kubernetes clusters that run within an existing cluster (also known as a host cluster) and share select resources. They are lightweight, cost-effective, secure, and resource efficient. Nested Clusters enable the rapid creation of securely-isolated environments for application testing and development without the infrastructure and operational overhead typically entailed by additional Kubernetes clusters.
```
Comment (tiwarisumit3) [inline content/docs/04-clusters/4-nested-clusters.md:19]: nit pick: share selected resources?

File: content/docs/04-clusters/4-nested-clusters/nested-cluster-quickstart.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+To access a Palette Sandbox, go to [SpectroCloud](https://spectrocloud.com) <!--confirm link--> and use your Github or Google SSO. You will be prompted to create a Palette account.
+You are allowed to deploy a managed Nested Cluster. See below under [Deploying a Nested Cluster](/clusters/nested-clusters/cluster-quickstart#deployinganestedcluster), follow the steps below under 
+
+
+# Prerequisites
+
+- You will need to have access to a Palette Account. Refer to the [Getting Started with Palette](/getting-started).
+
+
+- Ensure you have a functioning cluster already configure. Refer to the [New Clusters](/clusters/new-clusters) page for more information.
```
Comment (tiwarisumit3) [inline content/docs/04-clusters/4-nested-clusters/nested-cluster-quickstart.md:29]: 1. Enable the nested cluster option needs to be enabled from the settings tab.

2. Cluster should be launched with this settings.

File: content/docs/04-clusters/4-nested-clusters.md [hunk 3]
```diff
@@ -16,24 +16,25 @@ import Tooltip from "shared/components/ui/Tooltip";
...
-- LoadBalancer
-  - The host cluster must support dynamic provisioning of load balancers - either via a Cloud Controller Manager in the public cloud, or a bare metal load balancer provider such as MetalLB.
+- **LoadBalancer**
+  - The host cluster must support dynamic provisioning of load balancers, either via a Cloud Controller Manager in the public cloud or a bare metal load balancer provider, such as MetalLB.
 
-- Ingress
-  - The NGINX Ingress Controller must be deployed on the host cluster with SSL pass-through enabled. This allows TLS termination to occur at the nested cluster's Kubernetes API server.
-  - A wildcard DNS record must be configured that maps to the load balancer associated with the NGINX Ingress Controller, e.g., *.nested.host.1.spectrocloud.com.
+- **Ingress**
+  - The NGINX Ingress Controller must be deployed on the Host cluster with the SSL pass through enabled. This allows TLS termination to occur at the Nested Cluster's Kubernetes API server.
```
Comment (TylerGillson) [inline content/docs/04-clusters/4-nested-clusters.md:35]: should be: "with SSL passthrough enabled"

File: content/docs/04-clusters/4-nested-clusters/nested-cluster-quickstart.md [hunk 4]
```diff
@@ -15,113 +15,134 @@ import Tooltip from "shared/components/ui/Tooltip";
...
+- Ensure you have a functioning [cluster](/clusters/new-clusters) already configured. During the cluster creation, you will be able to configure it as a Host Cluster.
 
-- Ensure you have a functioning cluster already configure. Refer to the [New Clusters](/clusters/new-clusters) page for more information.
+  **Hosted Clusters**
 
+  There are three ways to engage with a Nested Cluster on Palette.
 
-# Deploying a Nested Cluster
-1. From the slide menu, select the **Clusters** tab and click the **Nested Clusters** tab to list the available Nested Clusters, and then select **Add New Nested Cluster**.
+   1. When you [create](/clusters/new-clusters) and [deploy](/clusters/nested-clusters/cluster-quickstart#deployinganestedcluster) a new Host Cluster, there is an option to **Enable Nested Clusters**. Refer to the [Cluster Endpoint](/clusters/nested-clusters/cluster-quickstart#loadbalancer) step below for more information on how to configure this option. The same settings listed below are available, when you are enabling a Nested Cluster and when deploying a Host Cluster deployment wizard.
```
Comment (TylerGillson) [inline content/docs/04-clusters/4-nested-clusters/nested-cluster-quickstart.md:33]: "below are available, when you are enabling" - no comma needed. Should be "below are available when you are enabling"

File: content/docs/04-clusters/4-nested-clusters/nested-cluster-quickstart.md [hunk 5]
```diff
@@ -15,113 +15,134 @@ import Tooltip from "shared/components/ui/Tooltip";
...
 
+    ![HostCluster](create-host-cluster.png "#width=600px")
 
-2. Select Deploy New Cluster or Import Cluster if you have your own cluster. For this exercise we will deploy a new cluster.
+   2. Similarly, you can [enable](/clusters/nested-clusters/cluster-quickstart#enablinganestedclusteronahostcluster) an existing cluster and configure it to become a Nested Cluster.
```
Comment (TylerGillson) [inline content/docs/04-clusters/4-nested-clusters/nested-cluster-quickstart.md:37]: "an existing cluster and configure it to become a Nested Cluster." --> "an existing cluster and allow it to host Nested Clusters."

File: content/docs/04-clusters/4-nested-clusters/nested-cluster-quickstart.md [hunk 6]
```diff
@@ -15,113 +15,134 @@ import Tooltip from "shared/components/ui/Tooltip";
...
 
+   3. Or, you can import a [Brownfield Cluster](/clusters/brownfield-clusters#importingabrownfieldcluster) and then, enable the Nested Cluster.
```
Comment (TylerGillson) [inline content/docs/04-clusters/4-nested-clusters/nested-cluster-quickstart.md:39]: "and then, enable the Nested Cluster." --> "and then enable it to host Nested Clusters."

File: content/docs/04-clusters/4-nested-clusters/nested-cluster-quickstart.md [hunk 7]
```diff
@@ -15,113 +15,134 @@ import Tooltip from "shared/components/ui/Tooltip";
...
+      **Note**: Use lowercase letters and do not add spaces.
+
+    - Provide a Description and Tags. These are optional.
+
+    - Click the **Attach Profile** button to assign a profile.
+
+      **Note**: Optionally, attach one or more Add-on layers to this cluster. If you do not have a Cluster Profile, see the [Creating Cluster Profile](/cluster-profiles/task-define-profile) page for more information.
+
+
+3. Set the **Quota Limits** for the following hardware settings:
```
Comment (TylerGillson) [inline content/docs/04-clusters/4-nested-clusters/nested-cluster-quickstart.md:123]: As of last night, quotas are being removed for 2.8

**Review by truegilb — CHANGES_REQUESTED** [review]
Overall looks great - as mentioned please remove quota from the description as it's the latest update that needs to happen in 2.8.

File: content/docs/04-clusters/4-nested-clusters/nested-cluster-quickstart.md [hunk 8]
```diff
@@ -0,0 +1,148 @@
...
+  - If **Ingress** is selected, a **Host Pattern** must be specified for this Host Cluster. To create a valid Host Pattern, the NGINX Ingress Controller must be deployed on the Host Cluster with SSL passthrough enabled. This allows TLS termination to occur at the nested cluster's Kubernetes API server.
+
+   Additionally, a wildcard DNS record must be configured that maps the Host Pattern to the load balancer associated with the NGINX Ingress Controller.
+
+   For example `*.nested.host.1.spectrocloud.com.`
+
+<InfoBox>
+<b>Recap</b>
+
+- Deploy the NGINX Ingress Controller on the Host Cluster and ensure that the SSL passthrough is enabled in the NGINX Ingress Controller pack's values.yaml. Specifically, charts.ingress-nginx.controller.extraArgs must be set as follows:
```
Comment (TylerGillson) [inline content/docs/04-clusters/4-nested-clusters/nested-cluster-quickstart.md:76]: should be "ensure that SSL passthrough is enabled"

**Review by TylerGillson — CHANGES_REQUESTED** [review]

**Review by tiwarisumit3 — APPROVED** [review]

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/854#" title="Author self-approved">Siboney1</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/854#pullrequestreview-1092564149" title="Approved">tiwarisumit3</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by tiwarisumit3** [comment]
Apart from minor comments from Tyler to be addressed, overall it looks good to me. Thanks Dorris.

**Review by truegilb — APPROVED** [review]

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/854#" title="Author self-approved">Siboney1</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/854#pullrequestreview-1092564149" title="Approved">tiwarisumit3</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/854#pullrequestreview-1093910754" title="Approved">truegilb</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by TylerGillson — APPROVED** [review]

**Pull request merged**